setup.py

```diff
@@ -1,40 +1,28 @@
-import re
 import setuptools
-import sys
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-
-with open("README.md", "r") as fh:
-    long_description = fh.read()
-
-with open('uwg/__init__.py', 'r') as fd:
-    version = re.search(
-        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-        fd.read(),
-        re.MULTILINE
-    ).group(1)
-
-try:
-    from semantic_release import setup_hook
-    setup_hook(sys.argv)
-except ImportError:
-    pass
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
 
 setuptools.setup(
     name="uwg",
-    version=version,
-    author="ladybug-tools",
-    description="The Urban Weather Generator engine for Urban Heat Island modelling",
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    author="Ladybug Tools",
+    author_email="info@ladybug.tools",
+    description="The Urban Weather Generator (uwg) is a Python application for modeling the urban heat island effect.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/ladybug-tools/uwg",
-    packages=['uwg'],
-    include_package_data=True,
+    packages=setuptools.find_packages(exclude=["tests*"]),
+    install_requires=requirements,
```

Comment:
You need to set `include_package_data=True` like you see here:
https://github.com/ladybug-tools/honeybee-standards/blob/master/setup.py#L18